### ISSUE TITLE
Login page in Chrome have constant refresh / reloading loop

### ISSUE THREAD

**Issue by AgelxNash** [evolution-cms/evolution#889]
In Chrome I also still have that constant refresh / reloading loop.
```
https://screencast-o-matic.com/watch/cFXQhQYR8U
```
I am totally stumped. Something is broken but not sure how to fix.

Any suggestions / ideas?

Thanks.

_Originally posted by @nick0 in https://github.com/evolution-cms/evolution/issues/886#issuecomment-437506100_

**Comment by nick0** [comment]
I think #890 and #889 are definitely related to #886.
The problems only occurred after taking advice in that thread to try the ElementsInTree plugin from the 1.4.x archive. That's when this issue started.
But lets try one by one.

All cache and history etc in all browsers are cleared after closing each time.

- Firefox 63 - in privacy or normal modes: no login page refresh / reload loop & can login normally
- Opera 56 - in normal mode: no login page refresh / reload loop & can login normally
- IE 11 - in normal mode: no login page refresh / reload loop & can login normally
- Chrome 70
-- in incognito mode: no login page refresh / reload loop & can login normally
-- in normal mode: the login page refresh / reload loop persists
--- so it seems contained to Chrome normal mode

When i try and look into the console, the page loads so quickly that I can't find an error before reloading.

This has me stumped.

Any ideas or suggestions?

Thanks.

**Comment by AgelxNash** [comment]
> Chrome 70
> -- in normal mode: the login page refresh / reload loop persists
> --- so it seems contained to Chrome normal mode

Disable all plugins in the browser and try again.

**Comment by 64j** [comment]
Try to repeat the operation as in the video.

https://monosnap.com/file/9c0bFG8TT0ZcAsieoE6JWf47tM1vIk

**Comment by nick0** [comment]
Thanks @AgelxNash. I disabled all plugins in Chrome and found the page to load once as expected. Great suggestion and much appreciated. I narrowed it down to the Norton Security Toolbar which turned on by itself - coincidental timing.

Thanks for the link @64j. Unfortunately I was unable to view the video in 4 different browsers on a pc. It displays a msg for me "The media could not be loaded, either because the server or network failed or because the format is not supported." I am interested to see the video in case that helps the other issue but cannot in the current format. Many thanks though.

Thank to you both.
